rtdb-limit-child-nodes/extension.yaml

```diff
@@ -55,7 +55,7 @@ resources:
       runtime: nodejs10
       eventTrigger:
         eventType: providers/google.firebase.database/eventTypes/ref.create
-        resource: projects/_/instances/${param:DATABASE_INSTANCE}/refs/${param:NODE_PATH}/{messageid}
+        resource: projects/_/instances/${DATABASE_INSTANCE}/refs/${NODE_PATH}/{messageid}
```

Comment:
@russellwheatley @joehan this is what I had to change to get it to work

Reply:
Ahh, that makes a lot of sense! I'll take on the CLI bug this week to unblock the change